src/app.js

```diff
@@ -1,24 +1,137 @@
-/* eslint-disable import/extensions */
-import express from "express";
-import helmet from "helmet";
-import cors from "cors";
+const express = require("express");
+const path = require("path");
 
-import notFoundMiddleware from "./middleware/not-found.js";
-import errorHandlerMiddleware from "./middleware/error-handler.js";
-import healthcheckRoute from "./routes/healthcheck.js";
-import userRoutes from "./routes/users.js";
+require("dotenv/config");
 
 const app = express();
+const cors = require("cors");
 
-app.use(helmet());
-app.use(cors());
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+const expresssession = require("express-session");
+const MongoStore = require("connect-mongo");
 
+const swaggerUi = require("swagger-ui-express");
+const swaggerJSDoc = require("swagger-jsdoc");
+
+const mongoose = require("mongoose");
+const healthcheckRoute = require("./routes/healthcheck");
+
+const env = require("./configs/index");
+const logger = require("./middleware/logger");
+const userRoutes = require("./routes/users");
+const actionStepsRoutes = require("./routes/actionsteps");
+const weightRoutes = require("./routes/weight");
+const categoryRoutes = require("./routes/category");
+const potentialRoutes = require("./routes/potential");
+const costRoutes = require("./routes/cost");
+const timescaleRoutes = require("./routes/timescale");
+
+const url = process.env.MONGO_URL;
+
+const connect = mongoose.connect(url);
+connect.then(
+  () => {
+    console.log("connected Correctly");
+  },
+  (err) => {
+    console.error(err);
+  }
```

Comment:
The connection has been done in the server file. This is not necessary

```js
server.js

await connectDB(dbUrl);
logger.info("✌️  DB loaded and connected!");
```

Reply:
Fixed.